src/hooks/useMarketPrice.ts

```diff
@@ -14,3 +16,19 @@ export const useMarketPrice = ({ base, quote }: Props) => {
   if (!basePrice || !quotePrice) return 0;
   return basePrice / quotePrice;
 };
+
+/** Return the market price of the base token in quote */
+export const getMarketPrice = async (
```

Comment:
did this get added by mistake in this PR? Dont see it being utilized in this PR?

Reply:
It's used here : 
https://github.com/bancorprotocol/carbon-app/pull/962/files#diff-b73d5b4566a2904c8a252328240eeb3d404a359e588c81aa599b3c5cd3770828R129